doc/user_guide/variables.rst

```diff
@@ -71,6 +71,8 @@ They only contain relative paths and are supposed to be relative to the ``${CMAK
 
    This is set to ``lib``.
    On non-Debian distributions it could be set to ``libexec``.
+   Note that setting this variable to anything but ``lib`` or ``libexec`` will
+   not work as expected since the logic in the code expects one of these values.
```

Comment:
Instead of:

    since the logic in the code expects one of these values.

I would write it like:

    since tools like ``catkin_find`` will only consider these two locations.